modules/flex/metadata.yaml

```
@@ -114,6 +126,17 @@ spec:
         description: Key/Value pairs to be passed to the Dataflow job (as used in the template).
         varType: map(string)
         defaultValue: {}
+        connections:
+          - source:
+              source: github.com/terraform-google-modules/terraform-google-pubsub
+              version: ">= 2.0"
+            spec:
+              outputExpr: "{\"inputTopic\": uri}"
+          - source:
+              source: github.com/terraform-google-modules/terraform-google-bigquery
+              version: ">= 8.0"
+            spec:
+              outputExpr: "{\"outputTableSpec\": \"project:bigquery_dataset.table_names[0]\"}"
```

What is the intention for this outputExpr?

* `bigquery_dataset` doesn't have a `table_names` field?
* Could you use `tables_names[0]` which is present in the output?
* \"project:bigquery_dataset.table_names[0]\" will be treated as constant string. Can't we directrly use `tables_names[0]` and dataflow application can prepend `project:`. WDYT?

```suggestion
              outputExpr: "{\"outputTable\": table_names[0]}"
```

---

Removed.